### PR TITLE
Fix(UG): Update room params to be code blocks

### LINKED ISSUE #346: UG: Standardise Room params to use code formatting

### PR DESCRIPTION
**Pull request by DrWala**
## What this does
- Closes #346 

## How to test
1. cd to `docs/` folder of the project.
2. Run `bundle exec jekyll serve`.
3. Open the jekyll-hosted website in your browser (default address: http://127.0.0.1:4000).
4. Check that the UG has the changes listed above

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -319,10 +319,10 @@ Format: `oedit INDEX [r/ROOM_NUMBER] [t/ROOM_TYPE] [g/TAG]`
 
 
 Parameters:
-* [INDEX](#index) The index of the room to edit.
-* [ROOM_NUMBER](#room_number) The room number to change the room identified by [INDEX](#index) to.
-* [ROOM_TYPE](#room_type) The room type to change the room identified by [INDEX](#index) to.
-* [TAG](#tag) Optional tags that may be updated for the room identified by [INDEX](#index).
+* [`INDEX`](#index) The index of the room to edit.
+* [`ROOM_NUMBER`](#room_number) The room number to change the room identified by [INDEX](#index) to.
```
Comment (stein414) [inline docs/UserGuide.md:323]: The `INDEX` at the end of the sentence needs the `` too

**Review by stein414 — CHANGES_REQUESTED** [review]
Just a slight change needed.

File: docs/UserGuide.md (hunk 1 above)
Comment (DrWala) [reply docs/UserGuide.md]: Done, thanks.

**Review by benedictkhoomw — APPROVED** [review]
LGTM.

**Review by stein414 — APPROVED** [review]
LGTM.

**Pull request merged**